org.mixedrealitytoolkit.spatialmanipulation/Solvers/TapToPlace.cs

```diff
@@ -328,7 +328,7 @@ public void StartPlacement()
             // selected and then immediately unselected. If two calls occur within the
             // double click timeout, then return to prevent an immediate object state switch.
             // Also, check that time is no 0 to allow for auto start functionality.
-            if (Time.time != 0 && (Time.time - lastTimeClicked) < DoubleClickTimeout)
+            if (Time.time != 0 && (Time.time - lastTimeClicked) < DoubleClickTimeout && !placementRequested)
```

Comment:
Perhaps initializing `lastTimeClicked = float.MaxValue` might be a better fix.

Reply:
Cause theoretically if Time.time is < DoubleClickTimeout, the same bug would occur